lib/npg_warehouse/loader/run_info.pm

```diff
@@ -73,20 +69,30 @@ sub _copy_table {
   my $rs_wh = $self->schema_wh->resultset($self->_prefix . $table);
   while (my $row_hash = $rs_npg->next) {
     delete $row_hash->{id_user};
+    if ($table eq 'Run'){
+      for my $key (keys %{$row_hash}){
```

Comment:
Would it be more straightforward to copy three values we need into a temp hash ref and then reassign `$row_hash`?